bst_plugin/process_nst_import_evt_events.m

```diff
@@ -0,0 +1,288 @@
+function varargout = process_nst_import_csv_events( varargin )
```

Comment:
It should be "process_nst_import_evt_events(..."

Reply:
oops :)